### PR TITLE
Moved rows to songrow component and started supporting votes

### PR DESCRIPTION
**Pull request by yaron**

### CODE REVIEW

**Comment by robindrost** [comment]
Haha full stack

File: src/containers/SongRow/SongRow.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      songIndex: props.songIndex
+    }
+
+    this.votePlus = this.votePlus.bind(this);
+    this.voteMin = this.voteMin.bind(this);
+  }
+
+  votePlus() {
+    const song = this.state.song;
+    song.votes[0].score++;
```
Comment (dylancom) [inline src/containers/SongRow/SongRow.js]: Never mutate this.state directly, as calling setState() afterwards may replace the mutation you made. Treat this.state as if it were immutable.

File: src/containers/SongRow/SongRow.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    this.setState({
+      song: song
+    })
+  }
+
+  voteMin() {
+    const song = this.state.song;
+    song.votes[0].score--;
```
Comment (dylancom) [inline src/containers/SongRow/SongRow.js]: Never mutate this.state directly, as calling setState() afterwards may replace the mutation you made. Treat this.state as if it were immutable.

**Comment by dylancom** [comment]
Never mutate this.state directly, as calling setState() afterwards may replace the mutation you made. Treat this.state as if it were immutable.
Zie: https://facebook.github.io/react/docs/react-component.html#state
Heb even een extra commit toegevoegd.

**Comment by dylancom** [comment]
De JSON response geeft nog een error message: 
"Trying to get property of non-object".
Het komt door votes { score } in de query.

verder pr0pz voor de fullstack coding 👍

**Comment by yaron** [comment]
Backend bug gefixed en het gebruik van prevState iets aangepast omdat deze nu de hele song overschreef met "1" in de state.
Als hij goed is kan je hem mergen, dan ga ik de merge conflicts in de andere pull request fixen.

**Pull request merged**